ReleaseChecklist.md

```diff
@@ -1,24 +1,52 @@
-Checklist for making a release:
+## Checklist for making a release:
 
+### Requirements
+ - [ ] AppVeyor account
+ - [ ] Lauchpad (Ubuntu One) account
+ - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
+ - [ ] Readthedocs account, access to the Solidity project
+ - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
+
+### Pre-release
  - [ ] Ensure that a Github project exists for the release.
  - [ ] Check that all issues and pull requests from the Github project to be released are merged to ``develop``.
- - [ ] Create a commit in ``develop`` that updates the ``Changelog`` to include a release date (run ``./scripts/tests.sh`` to update the bug list). Sort the changelog entries alphabetically and correct any errors you notice.
+
+### Changelog
+ - [ ] Create a commit in ``develop`` that updates the ``Changelog`` to include a release date.
+ - [ ] Run ``./scripts/tests.sh`` to update the bug list.
+ - [ ] Sort the changelog entries alphabetically and correct any errors you notice.
```

Comment:
Should do that before actually creating the commit :)

Reply:
So it should be: sort, script, commit, pull request

Reply:
Also, perhaps don't mention "in `develop`" explicitly, sounds like you directly push to the develop branch.

Reply:
YES :)

Reply:
> Also, perhaps don't mention "in `develop`" explicitly, sounds like you directly push to the develop branch.

Sure. We had that discussion already.